zenoh-kotlin/src/commonMain/kotlin/io/zenoh/SessionInfo.kt

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright (c) 2023 ZettaScale Technology
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+//
+// Contributors:
+//   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+//
+
+package io.zenoh
+
+import io.zenoh.protocol.ZenohID
+
+/**
+ * Class allowing to obtain the information of a [Session].
+ */
+class SessionInfo(private val session: Session) {
```

Comment:
Why using `Session` instead of `SessionInfo`?

Reply:
I'll let @wyfo and @DariusIMP decide if this comment is a blocker for this PR. Otherwise it LGTM.